docs/pages/includes/desktop-access/desktop-config.yaml

```diff
@@ -27,10 +27,19 @@ windows_desktop_service:
     # You can skip LDAPS certificate verification by setting
     # this to true. It is recommended that this be set to false
     # and the certificate added your system's trusted repository,
-    # or its filepath provided in with the der_ca_file variable below.
+    # or its filepath provided in with der_ca_file variable below
+    # or as PEM encoded certificate using ldap_ca_cert variable.
     insecure_skip_verify: false
     # DER encoded LDAP CA certificate.
+    # deprecated: prefer ldap_ca_cert instead
```

Comment:
That's a great suggestion! Thanks! I've also moved the ldap_ca_cert key so it's now the first key after the insecure_skip_verify rather than the `der_ca_file`. If you're reading comments and following them one by one, you'll encounter the correct settings first. Settings that are deprecated will be the last.